src/pages/Home.js

```diff
@@ -2,16 +2,25 @@ import styled from 'styled-components';
 import Header from '../components/Header';
 import PosterList from '../components/PosterList';
 import Navigation from '../components/Navigation';
+import FetchError from '../components/FetchError';
+import useCheckFetch from '../hooks/useCheckFetch';
 
 export default function Home({ popularSeries, popularMovies }) {
+  const { data } = useCheckFetch(popularSeries, popularMovies);
+
   return (
     <Wrapper>
       <Header />
-      <main>
-        <PosterList list={popularSeries} listName={'Beliebte Serien'} />
-        <PosterList list={popularMovies} listName={'Beliebte Filme'} />
-      </main>
-      <Navigation />
+      {!data && <FetchError />}
+      {data && (
+        <>
+          <main>
+            <PosterList list={popularSeries} listName={'Beliebte Serien'} />
+            <PosterList list={popularMovies} listName={'Beliebte Filme'} />
+          </main>
+          <Navigation />
+        </>
+      )}
```

Comment:
What about a ternary operator here?
```suggestion
      {!data ? 
      <FetchError />
      : (
        <>
          <main>
            <PosterList list={popularSeries} listName={'Beliebte Serien'} />
            <PosterList list={popularMovies} listName={'Beliebte Filme'} />
          </main>
          <Navigation />
        </>
      )}
```
If you use it, check whether the brackets are correct this way.